healthcheck.sh

```diff
@@ -0,0 +1,20 @@
+#! /bin/bash
+
+r=`$KAFKA_HOME/bin/zookeeper-shell.sh $KAFKA_ZOOKEEPER_CONNECT <<< "ls /brokers/ids" | tail -1 | jq '.[]'`   
+ids=( $r )                                                                                         
```

Comment:
trailing whitespace on this line and the previous line

also fwiw, using "health☑️" as a branch name makes it difficult for others to use your branch.